openshift/templates/jenkins-persistent.json

```diff
@@ -326,6 +330,12 @@
       "displayName": "Fatal Error Log File",
       "description": "When a fatal error occurs, an error log is created with information and the state obtained at the time of the fatal error.",
       "value": "false"
+    },
+    {
+      "name": "JENKINS_UC_INSECURE",
+      "displayName": "Allows use of Jenkins Update Center repository with invalid SSL certificate",
+      "description": "Whether to allow use of a Jenkins Update Center that uses invalid certificate (self-signed, unknown CA). If any value other than 'false', certificate check is bypassed.",
```

Comment:
Add a note that the default is to enforce the certificate check

Reply:
Fixed